packages/core/src/binary-data/object-store/object-store.service.ee.ts

```diff
@@ -154,9 +145,7 @@ export class ObjectStoreService {
 			'Content-MD5': createHash('md5').update(body).digest('base64'),
 		};
 
-		const path = `${this.bucket.name}/?delete`;
-
-		return await this.request('POST', this.host, path, { headers, body });
+		return await this.request('POST', '?delete', { headers, body });
```

Comment:
Why are we passing query strings in the path, instead in the `qs` object? Because it doesn't support query stings with only key and not value?

Reply:
this was because `URL` can't generate strings like `http://domain/path?delete`. there will always be a `=` after the `delete`. which I think is fine. so I've updated this to use `qs`, and updated the tests accordingly.